README.md

```diff
@@ -55,7 +55,7 @@ Example RESTful interaction:
 | Set communication step in seconds                                     |  PUT ``step`` with argument ``step=<value>``              |
 | Receive sensor signal names (y) and metadata                          |  GET ``measurements``                                     |
 | Receive control signals names (u) and metadata                        |  GET ``inputs``                                           |
-| Receive test result data                                              |  GET ``results``                                          |
+| Receive test result data for the given point name between the start and final time in seconds |  PUT ``results`` with arguments ``point_name=<string>``, ``start_time=<value>``, ``final_time=<value>``|
```

Comment:
Is there a reason to use the "point" keyword? the keyword "variable" feels more intuitive to me. I see a point being a set of variable values at a certain time whereas here we're talking about time series. Does "point" belong to the BMS jargon? in that case I'd have nothing to say. If not, I'd suggest to change it to "variable".

Reply:
A "point" is BMS jargon for input/output variable.  Part of a BMS/Controls design is a "points list," which lists all of the sensor and control "points."

Reply:
Great, let's keep it like this then.

Reply:
Sounds good.  I added "point" to the readme description of /measurements and /inputs in  https://github.com/ibpsa/project1-boptest/pull/268/commits/d1c0eaa5353713ac3da28bf595b2259bd2aeb538.